src/coverlet.console/Logging/ConsoleLogger.cs

```diff
@@ -1,39 +1,39 @@
 using System;
-using static System.Console;
+using Coverlet.Core.Logging;
+using SystemConsole = System.Console;
 
 namespace Coverlet.Console.Logging
 {
     class ConsoleLogger : ILogger
     {
-        public void LogError(string message)
-        {
-            ForegroundColor = ConsoleColor.Red;
-            WriteLine(message);
-            ForegroundColor = ConsoleColor.White;
-        }
+        private static readonly object _sync = new object();
 
-        public void LogInformation(string message)
-        {
-            WriteLine(message);
-        }
+        public void LogError(string message) => WriteLine(message, ConsoleColor.Red);
 
-        public void LogSuccess(string message)
-        {
-            ForegroundColor = ConsoleColor.Green;
-            WriteLine(message);
-            ForegroundColor = ConsoleColor.White;
-        }
+        public void LogError(string message, Exception ex) => LogError($"{message}{Environment.NewLine}{ex}");
 
-        public void LogVerbose(string message)
-        {
-            throw new System.NotImplementedException();
-        }
+        public void LogInformation(string message) => WriteLine(message);
```

Comment:
You could call same helper with current `ConsoleColor` to be consistent with other methods

Reply:
never mind...defaul color,  missed that perfect

Reply:
Because we can change "default" foreground color
![image](https://user-images.githubusercontent.com/7894084/52882246-593be200-3167-11e9-806b-403ef4fff318.png)